esp/src/src-react/components/Title.tsx

```diff
@@ -119,18 +121,38 @@ export const DevTitle: React.FunctionComponent<DevTitleProps> = ({
         };
     }, [currentUser?.username]);
 
-    const btnStyles = mergeStyleSets({
+    const btnStyles = React.useMemo(() => mergeStyleSets({
         errorsWarnings: {
             border: "none",
             background: "transparent",
             minWidth: 48,
             padding: "0 10px 0 4px",
-            color: theme.semanticColors.link
+            color: logIconColor
         },
         errorsWarningsCount: {
             margin: "-3px 0 0 -3px"
         }
-    });
+    }), [logIconColor]);
+
+    React.useEffect(() => {
+        switch (log[0]?.level) {
+            case Level.critical:
+            case Level.emergency:
+            case Level.error:
+                setLogIconColor(theme.palette.red);
+                break;
+            case Level.warning:
+                setLogIconColor(theme.palette.yellow);
```

Comment:
Should use semantic colours

Reply:
This is what I am using in the toaster (and it should get commoned up in the future):
```
function iconColor(level: Level): string {
    switch (level) {
        case Level.info:
        case Level.notice:
            return semanticColors.infoIcon;
        case Level.warning:
            return semanticColors.warningIcon;
        case Level.error:
            return semanticColors.errorIcon;
        case Level.critical:
        case Level.alert:
        case Level.emergency:
            return semanticColors.severeWarningIcon;
        case Level.debug:
        default:
            return semanticColors.successIcon;
    }
}
```